src/helpers/figures/charts/waterfall_chart.ts

```diff
@@ -242,14 +242,25 @@ function getWaterfallConfiguration(
     labels: {
       generateLabels: () => {
         const legendValues = [
-          { text: _t("Positive values"), fontColor, fillStyle: positiveColor },
-          { text: _t("Negative values"), fontColor, fillStyle: negativeColor },
+          {
+            text: _t("Positive values"),
+            fontColor,
+            fillStyle: positiveColor,
+            strokeStyle: positiveColor,
+          },
+          {
+            text: _t("Negative values"),
+            fontColor,
+            fillStyle: negativeColor,
+            strokeStyle: negativeColor,
+          },
         ];
         if (chart.showSubTotals || chart.firstValueAsSubtotal) {
           legendValues.push({
             text: _t("Subtotals"),
             fontColor,
             fillStyle: subTotalColor,
+            strokeStyle: subTotalColor,
```

Comment:
Do these changes have something to do with the new colors ? Or was somehting broken before ?

Reply:
![image](https://github.com/user-attachments/assets/eea0137b-0b1a-414e-a971-1307a81305e9)
This has beed added to avoid the gray border here